.travis.yml

```diff
@@ -24,8 +24,8 @@ before_script:
  - openssl req -x509 -newkey rsa:4096 -keyout server.key -out server.crt -days 365 -nodes -subj /CN=localhost
 
 script:
- - ./travis-install-kafka.sh
- - ./travis-test.sh
+ - ./scripts/install-local-kafka.sh
```

Comment:
this is linked to as travis-install-kafka from the INSTALL instructions iirc, pls could you have a grep and fix